# LD robot related params
arcl_api_server:
  ros_parameters:
    ip_address: "192.168.48.69"
    port: 7171
    def_arcl_passwd: "omron"

# Local machine related params
ld_param_node:
  ros_parameters:
    local_ip: "192.168.48.78"
    local_port: 7179
